Gemfile

```diff
@@ -98,6 +98,7 @@ gem "savon",                          "~>2.2.0",   :require => false  # Automate
 gem "snmp",                           "~>1.2.0",   :require => false
 gem "uglifier",                       "~>2.7.1",   :require => false
 gem "sshkey",                         "~>1.8.0",   :require => false
+gem "fog-vcloud-director",            "~>0.1.0",   :require => false
```

Comment:
@durandom Did you want a way to track dependencies of providers in the Gemfile?  Like a group or anything?  Just trying to find a way to make that part of the extraction process easier.